.github/policies/issue.flag-for-triage.yml

```diff
@@ -15,5 +16,37 @@ configuration:
       then:
       - addLabel:
           label: 'Needs: Triage (Functions)'
+
+    - description: Add needs attention label to reopened issues
+      if:
+      - payloadType: Issues
+      - and:
+        - isAction:
+            action: Reopened
+        - not:
+            hasLabel: 'Needs: Attention :wave:'
+      then:
+      - addLabel:
+          label: 'Needs: Attention :wave:'
```

Comment:
We had a few instances of things being in an improper state because they were closed and reopened. This should allow us to more easily catch theses.